domains/lucas-zick.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "LucasZick",
+        "email": "lucasalexandrezick@gmail.com"
+    },
+    "record": {
+        "A": ["144.22.193.227"]
```

Comment:
I cannot reach your website
> 144.22.193.227 took too long to respond.

Reply:
Hello, I believe it might be because the website is running on HTTPS. I wasn't sure what to add to make it explicit, as using the IP without HTTPS doesn't load anything. Can you clarify for me what needs to be changed or if this is indeed the issue?

Reply:
By updating, I solved the problem, I had port 80 closed, which blocked the website from redirecting, now it is redirecting the HTTP request to HTTPS. now it probably works

Reply:
I will open a new PR